src/components/screens/ExperimentCreator.vue

```diff
@@ -0,0 +1,1348 @@
+                      </label>
+                    </div>
+                    <div class="mavedb-wizard-content field">
+                      <span class="p-float-label">
+                        <FileUpload
+                            :id="$scopedId('input-extraMetadataFile')"
+                            :auto="false"
+                            chooseLabel="Extra metadata"
+                            :class="inputClasses.extraMetadataFile"
+                            :customUpload="true"
+                            :fileLimit="1"
+                            :showCancelButton="false"
+                            :showUploadButton="false"
+                            @remove="fileCleared('extraMetadataFile')"
+                            @select="fileSelected('extraMetadataFile', $event)"
+                        >
+                          <template #empty>
+                            <p>Drop a JSON file here.</p>
+                          </template>
+                        </FileUpload>
+                      </span>
+                      <span v-if="validationErrors.extraMetadata" class="mave-field-error">{{validationErrors.extraMetadata}}</span>
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <div class="mavedb-wizard-step-controls-row">
+                <div class="flex justify-content-end mavedb-wizard-step-controls pt-4">
+                  <Button label="Next" :disabled="this.maxWizardStepValidated < activeWizardStep" icon="pi pi-arrow-right" iconPos="right" @click="showNextWizardStepIfValid(showNextWizardStep)" />
+                </div>
+              </div>
+            </template>
+          </StepperPanel>
+          <StepperPanel v-if="itemStatus == 'NotLoaded' || this.item.private" header="Keywords">
+            <template #header="{index, clickCallback}">
+              <button class="p-stepper-action" :disabled="maxWizardStepEntered < index || maxWizardStepValidated < index - 1" role="tab" @click="clickCallback">
+                <span class="p-stepper-number">{{ index + 1 }}</span>
+                <span class="p-stepper-title">Keywords</span>
+              </button>
+            </template>
+            <template #content="{prevCallback: showPreviousWizardStep}">
+              <div class="mavedb-wizard-row">
+                <div class="mavedb-wizard-help">
+                  <label>Choose related keywords(Optional).</label>
+                  <div class="mavedb-help-small">
+                    Experiments can be tagged with optional keywords.
+                    In a future release, the keyword vocabulary will become restricted and keyword selection will be mandatory. 
+                  </div>
+                </div>
+                <div class="mavedb-wizard-content field">
+                  <div class="keyword-editor">
+                    <div v-for="keyword in keywordData" :key="keyword.key">
```

Comment:
I know you spent a lot of time on the keyword UI (and I think it looks good), but might it make sense to have a row for each of the keyword categories with help text on the left now that we are putting this in the wizard? Space was more limited in the previous upload flow than it is now. 

We could even have input switches like in the score set wizard that would help to indicate that these fields are optional. Users would select that yes they want to add a keyword for a specific vocabulary. I'm not wedded to that though, since it does add significantly more clicks.

Reply:
No worries! This part was originally used in viewing experiment page. Ashley suggested me to add them in creating new experiment too so they're in here. I'll move them to the left help text. After confirming the final keywords list, all of them will be compulsory so I think we can simplify the optional effect first.

Reply:
Sorry, I think keep these description in small information is better because all of these keywords are in a for loop. If users choose either "endogenous locus library method" or "in vitro construct library method", new keyword options will pop out. The layout is very ugly and they don't match as a row orderly. We can adjust this part when we get the final list cause at that time the keywords' relationships may change.